app-client/src/main/java/com/keycloak/cdc/restcontroller/AppRestController.java

```diff
@@ -0,0 +1,57 @@
+package com.keycloak.cdc.restcontroller;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.servlet.mvc.method.annotation.SseEmitter;
+
+import com.keycloak.cdc.service.AppService;
+
+@RestController
+public class AppRestController {
+
+	@Value("${users.url}")
+	String GET_USERS_URL;
+	
+	@Autowired
+	AppService service;
+	
+	@Autowired
+	private RestTemplate restTemplate;
+
+	final List<SseEmitter> emitters = new CopyOnWriteArrayList<>();
+
+	@GetMapping("/users")
+	public ResponseEntity<String> getUsers() {
+		
+		HttpHeaders headers = new HttpHeaders();
+		headers.setAccept(Arrays.asList(MediaType.APPLICATION_JSON));
+		HttpEntity<String> entity = new HttpEntity<String>("parameters", headers);
+		ResponseEntity<String> users = restTemplate.exchange(GET_USERS_URL, HttpMethod.GET, entity, String.class);
+		System.out.println(users.getBody());
+		return users;
+	}
+	
+	@GetMapping("/notification")
+	public ResponseEntity<SseEmitter> doNotify() throws InterruptedException, IOException {
+		final SseEmitter emitter = new SseEmitter();
+		service.addEmitter(emitter);
+		service.doNotify();
+		emitter.onCompletion(() -> service.removeEmitter(emitter));
+		emitter.onTimeout(() -> service.removeEmitter(emitter));
+		return new ResponseEntity<>(emitter, HttpStatus.OK);
+	}
```

Comment:
After going through the whole project, I do not see where this is used. Kindly review

Reply:
Was actually planning on removing it.
Resolved!